TRRandomizerView/Utilities/ControlUtils.cs

```diff
@@ -8,94 +8,93 @@
 
-        public static class DefaultIcons
-        {
-            private static Icon _largeFolderIcon, _smallFolderIcon;
+        return wpfBitmap;
+    }
 
-            public static Icon FolderLarge => _largeFolderIcon ?? (_largeFolderIcon = GetStockIcon(SHSIID_FOLDER, SHGSI_LARGEICON));
-            public static Icon FolderSmall => _smallFolderIcon ?? (_smallFolderIcon = GetStockIcon(SHSIID_FOLDER, SHGSI_SMALLICON));
+    public static class DefaultIcons
+    {
+        private static Icon _largeFolderIcon, _smallFolderIcon;
 
-            [DllImport("shell32.dll")]
-            private static extern int SHGetStockIconInfo(uint siid, uint uFlags, ref SHSTOCKICONINFO psii);
+        public static Icon FolderLarge => _largeFolderIcon ??= GetStockIcon(SHSIID_FOLDER, SHGSI_LARGEICON);
+        public static Icon FolderSmall => _smallFolderIcon ??= GetStockIcon(SHSIID_FOLDER, SHGSI_SMALLICON);
 
-            [DllImport("user32.dll")]
-            private static extern bool DestroyIcon(IntPtr handle);
+        [DllImport("shell32.dll")]
+        private static extern int SHGetStockIconInfo(uint siid, uint uFlags, ref SHSTOCKICONINFO psii);
 
-            [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
-            private struct SHSTOCKICONINFO
-            {
-                public uint cbSize;
-                public IntPtr hIcon;
-                public int iSysIconIndex;
-                public int iIcon;
-                [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-                public string szPath;
-            }
+        [DllImport("user32.dll")]
+        private static extern bool DestroyIcon(IntPtr handle);
 
-            private const uint SHSIID_FOLDER = 0x3;
-            private const uint SHGSI_ICON = 0x100;
-            private const uint SHGSI_LARGEICON = 0x0;
-            private const uint SHGSI_SMALLICON = 0x1;
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        private struct SHSTOCKICONINFO
+        {
+            public uint cbSize;
+            public IntPtr hIcon;
+            public int iSysIconIndex;
+            public int iIcon;
+            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
+            public string szPath;
+        }
 
-            private static Icon GetStockIcon(uint type, uint size)
-            {
-                SHSTOCKICONINFO info = new SHSTOCKICONINFO();
-                info.cbSize = (uint)Marshal.SizeOf(info);
+        private const uint SHSIID_FOLDER = 0x3;
+        private const uint SHGSI_ICON = 0x100;
+        private const uint SHGSI_LARGEICON = 0x0;
+        private const uint SHGSI_SMALLICON = 0x1;
+
+        private static Icon GetStockIcon(uint type, uint size)
+        {
+            SHSTOCKICONINFO info = new();
+            info.cbSize = (uint)Marshal.SizeOf(info);
 
-                SHGetStockIconInfo(type, SHGSI_ICON | size, ref info);
+            _ = SHGetStockIconInfo(type, SHGSI_ICON | size, ref info);
```

Comment:
Does the IDE warn about this kind of thing as well?

Reply:
It does warn, we just discard the native result here. I'd like to move to using custom icons instead of having to pull in the ones from the OS, just so we can eliminate this class. Another future task will be revamping the UI so we can do something cleaner then.